src/ExportObject.cs

```diff
@@ -148,6 +154,92 @@ public virtual void HandleMethodCall (MessageContainer method_call)
 			}
 		}
 
+		private void HandlePropertyCall(MessageContainer method_call)
+		{
+			Message msg = method_call.Message;
+			MessageReader msgReader = new MessageReader(msg);
+			MessageWriter retWriter = new MessageWriter();
+
+			object[] args = MessageHelper.GetDynamicValues(msg);
+
+			string face = (string)args[0];
+
+			if ("GetAll" == method_call.Member) {
+				conn.MaybeSendUnknownMethodError(method_call);
+				return;
+			}
+
+			string name = (string)args[1];
+
+			PropertyInfo pi = Object.GetType().GetProperty(name);
+			MethodInfo mi;
+			MethodCaller pc;
+			Signature outSig, inSig = method_call.Signature;
+
+			switch (method_call.Member) {
+				case "Set":
+					mi = pi.GetSetMethod ();
+					pc = TypeImplementer.GenSetCall(pi);
+					outSig = Signature.Empty;
+					break;
+				case "Get":
+					mi = pi.GetGetMethod ();
+					pc = TypeImplementer.GenGetCall(pi);
```

Comment:
TODO: Cache the result of delegate generation. I used a dedicated class on my master branch for this but it seemed a bit overkill when I revisited the patch.